plugins/search-backend-module-explore/src/alpha.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @packageDocumentation
+ * A module for the search backend that exports Explore modules.
+ */
+
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { TaskScheduleDefinition } from '@backstage/backend-tasks';
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import { searchIndexRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
+
+import {
+  ToolDocument,
+  ToolDocumentCollatorFactory,
+  ToolDocumentCollatorFactoryOptions,
```

Comment:
You'll need to import this as `import { ToolDocumentCollatorFactoryOptions } from '@backstage/plugin-search-backend-module-explore'`